addons/google_calendar/models/google_sync.py

```diff
@@ -130,7 +126,6 @@ def _from_google_ids(self, google_ids):
         return self.browse(self._event_ids_from_google_ids(google_ids))
 
     @api.model
-    @ormcache_context('google_ids', keys=('active_test',))
     def _event_ids_from_google_ids(self, google_ids):
         return self.search([('google_id', 'in', google_ids)]).ids
```

Comment:
Actually you can inline this line inside `_from_google_ids` (if there is no more ormcache, it doesn't make sense to have this indirection IMO).